Chapter_Version_Control.md

```diff
@@ -1,552 +1,619 @@
+**Pull request:** Proposed changes to a remote repo. Collaborators without write access can send a pull request to the administrator with the changes they've made to the repo. The administrator can then approve and merge or reject the changes to the main repository. For open source projects pull requests can be sent by anyone that has forked a project.
 
---------
+**Push:** Sending changes to a remote repo. The remote repository is updated with the changes pushed and now mirrors the local repo.
 
-From [here](http://crlionline.net/node/198). **No license but active, could reach out**
+**Repository:** Refers to a project folder that is being tracked by Git and containing project files. Also called 'repo' for short they can be local as well as hosted on GitHub.
 
-If you are not ready for this yet, you will be at some time - when you have felt the horror of loosing days if not weeks of work. But versioning is useful for less dramatic purposes as well, not only for backup. For instance, to help you understand why in the past you made certain changes, why you did a certain analysis in the way you did it. Or you simply want to undo a change in your analysis. Or because you want to keep an audit trail of your analysis. Or to share it with others.
+**SHA:** Unique string of numbers of letters used to identify every commit or node in the repository.
 
-You yourself understand what the saved file does, even weeks or months later;
-Others can easier understand what you did. This is helpful should you want to share your analysis (not only your data), and/or make it auditable--more generally, reproducible, which is good scientific practice.
-This has the additional benefit that you need to understand for yourself what it is you just did.
+**Staged:** Changes that will be included in the next commit.
 
-When you restore from a backup, you are going back in time: I want to restore the last version, or to yesterday's (last week's) version. When you restore from a versioning system, you go back to a decision: I want to restore my former version, or an alternative version (a different branch),  because that one turns out to be the better solution or approach. Backup is a special case of versioning.
+## What is version control?
 
-There are numerous tools for versioning, and the best know one is git and its web-based version, Github. (They are not really strongly different, in that you can use GitHub as the remote repository for your local git repository).
+What is “version control”, and why should you care? Version control is a system that records changes to a file or set of files over time so that you can recall specific versions later. It is typically applied to managing changes in code, though in reality you can do this with nearly any type of file on a computer.
```

Comment:
I would like to add something like that in, but you're right, it wouldn't fit best here. Let's resolve this comment, and we can see if there's anywhere it would fit more maturally later.